radix-engine/src/updates/cuttlefish.rs

```diff
@@ -130,3 +144,28 @@ fn generate_cuttlefish_transaction_validation_updates() -> StateUpdates {
         ),
     )
 }
+
+fn generate_cuttlefish_update_min_rounds_per_epoch<S: SubstateDatabase + ?Sized>(
+    db: &S,
+) -> StateUpdates {
+    let mut consensus_manager_config = db
+        .get_existing_substate::<FieldSubstate<VersionedConsensusManagerConfiguration>>(
+            CONSENSUS_MANAGER,
+            MAIN_BASE_PARTITION,
+            ConsensusManagerField::Configuration,
+        )
+        .into_payload()
+        .fully_update_and_into_latest_version();
+
+    consensus_manager_config
+        .config
+        .epoch_change_condition
+        .min_round_count = 100;
```

Comment:
So on the node, in `GenesisManagerConfig`, I see two main variants: `productionDefaults()` (used by Mainnet and Stokenet) sets `minRoundsPerEpoch` to 500, but `testDefaults()` already has it on 100, and other networks have it set to `Long.MAX_VALUE`.

Could we be clever here and only update it to `100` if it's currently `500`? This avoids breaking test networks which configure genesis in a certain way and then rely on network lengths being a certain length.

Reply:
In the scenarios I suggest we can then override the `BabylonSettings` to use the mainnet defaults, so we can observe this change there 👍. The mainnet genesis configuration was as follows. This will cause a change in the scenario summary for genesis, but that's fine :)

```
        ConsensusManagerConfig {
            max_validators: 100,
            epoch_change_condition: EpochChangeCondition {
                min_round_count: 500,
                max_round_count: 3000,
                target_duration_millis: 300000,
            },
            num_unstake_epochs: 2016,
            total_emission_xrd_per_epoch: dec!(2853.881278538812785),
            min_validator_reliability: Decimal::one(),
            num_owner_stake_units_unlock_epochs: 8064,
            num_fee_increase_delay_epochs: 4032,
            validator_creation_usd_cost: dec!(1000),
        }
```

Reply:
> Could we be clever here and only update it to 100 if it's currently 500? This avoids breaking test networks which configure genesis in a certain way and then rely on network lengths being a certain length.

Yup, we could do it that way. I think that we could also use the feature you added a while back where protocol updates can have settings to control what the min rounds per epoch gets updated to. How would you feel about that? 

>In the scenarios I suggest we can then override the BabylonSettings to use the mainnet defaults, so we can observe this change there 👍. The mainnet genesis configuration was as follows. This will cause a change in the scenario summary for genesis, but that's fine :)

Sounds good, will do!

Reply:
> I think that we could also use the feature you added a while back where protocol updates can have settings to control what the min rounds per epoch gets updated to. How would you feel about that?

Yep, it was designed for this. Happy to have a setting here, but the setting should default to "automatic" which should follow the logic specified I think :). Basically we just want to be changing the production default.